src/reducers/__tests__/players.js

```diff
@@ -0,0 +1,96 @@
+import {ActionTypes, CellStates} from '../../constants';
+import reducer from '../players.js';
+
+const {INITIAL_STATE, BOARD_TILE_SELECTED, RACK_TILE_SELECTED, END_TURN} = ActionTypes;
+
+describe('src/reducers/players', function() {
```

Comment:
it's probably going to be easier to just test one of `player` and `players`.  you aren't stubbing anything, so testing `players` should be sufficient.  or, you can test both but omit the cases from `players` that are already tested in `player`.